src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -1,4 +1,5 @@
 import { CertifyIF, DefineCompanyIF, NameRequestIF, TombStoneIF } from '@/interfaces'
+import { PeopleAndRoleIF } from '../stepper-interfaces/AddPeopleAndRole/add-people-role-interface'
```

Comment:
Not 100% sure but i believe this could be added to the above line. If exported correctly from its sub directory i think all the interfaces should be accessible from '@/interfaces'.